sdk/nodejs/src/utils/userBucketingHelper.ts

```diff
@@ -7,3 +7,26 @@ export function bucketUserForConfig(user: DVCPopulatedUser, token: string): Buck
         getBucketingLib().generateBucketedConfigForUser(token, JSON.stringify(user))
     ) as BucketedUserConfig
 }
+
+export function getVariableTypeCode(type: VariableType): number {
+    // TODO: can't figure out how to get the enum values from the bucketing lib as its dynamically imported
```

Comment:
@ajwootto any idea how to handle this? I tried loading the enum from the `Exports` of `@devcycle/bucketing-assembly-script` and from the `getBucketingLib()` object itself that is loaded with no luck.

Reply:
it gets compiled to this: 
<img width="1510" alt="image" src="https://user-images.githubusercontent.com/1219069/225021811-6241a9bc-5f7d-4052-9328-29df1f2044ae.png">

Reply:
you can maybe look at how I did it in Go. They were available as globals named `VariableType.Boolean` etc.